docs/src/components/version-switcher.tsx

```diff
@@ -44,7 +44,6 @@ export default function VersionSwitcher(): JSX.Element {
   return (
     versions.length > 0 && (
       <DropdownNavbarItem
-        className="navbar__item"
```

Comment:
To avoid issues in the future, this line could be added back but set to a custom value that is very unlikely to be used elsewhere:

```suggestion
        className="version-switcher-34ab39"
```

I came up with `34ab39` just picking letters and numbers on my keyboard.

The CSS would need to be updated accordingly.